### ISSUE TITLE
"Next" buttons have no focus indicator

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#783]
## What
Next buttons have no focus indicator, so keyboard users won't know when they're focused on them.
![screenshot of the Next button](https://user-images.githubusercontent.com/169471/63117587-faf28f00-bf50-11e9-841a-466f19a8b58c.png)

![screenshot of dev tools showing 'outline:none' in the button's focus state](https://user-images.githubusercontent.com/169471/63117655-15c50380-bf51-11e9-8460-52630539eeff.png)

[All interactive elements must have a focus indicator](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html), so the CSS `outline:none` rule needs to be removed for these elements.

## When
ASAP please.
